### ISSUE TITLE
status not changed from OK

### ISSUE THREAD

**Issue by maximnik** [scobal/seyren#275]
I've created several checks with API requests>
Example:
{
  "name":"<name>",
  "target":"<target>",
  "warn":"1",
  "error":"0",
  "enabled":true,
  "live":true,
  "from":"3h",
  "until":"now"
}

I can  see all metrics in dashboard.

I need alert when value is below 1 for certain time. But status for check is always OK.

I've tried to changed Warn And Critical values to 0.25 and 0.5 accordingly, but status still is OK. And no alert. I even change From to 5 min (Until: now). But still nothing happen.

**Comment by neilprosser** [comment]
I see you've added `live: true` to your check. Are you using the pickle functionality? I don't use it and just have Seyren query Graphite via the `/render` API so all my checks have `live: false`.

**Comment by maximnik** [comment]
I've removed "live" from api check creation.
Now it looks so:
{
  "name":"check_customer_${a}_status_success",
  "target":"sent eprocessor.${db}.feeds_per_customer.${a}.sent.success",
  "warn":1",
  "error":"0",
  "enabled":true,
  "from":"3h",
  "until":"now"
}

Still status OK only and Doesn't change to WARN or Critical.

**Comment by neilprosser** [comment]
Strange. Are those `${a}` type variables actually included in the check name and target? Also, you've got a space between 'sent' and 'eprocessor'. That's not going to be a valid Graphite target.

Do you see graphs display when you view the check in Seyren?

Are you able to provide the output of Seyren's `/checks` resource after you've created the check? You'll need the ID of the check and will want to call `http://<seyren-host>:<seyren-port>/seyren/api/checks/{check-id}` (or possibly drop the `/seyren` if you're not using a context-root).

**Comment by rsetzer** [comment]
We have Graphite setup and working and have built several dashboards presenting metrics gleaned from several applications. We have added Seyren to the picture and are attempting to create several checks that we would like to examine data points within a select window of time i.e., 4:00AM - 6:00AM daily at a specific time of day i.e., 6:05AM.

I have been attempting to determine exactly what the From and Until properties in the Create Check UI on Seyren's dashboard do exactly. I have tried several different formats for entering time to no avail. Every time I do so the check quits working until I delete and re-add the check. 

What exactly do these properties control (data points window or check execution window) and what format is expected?
